cmd/scaler/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.8
+
+COPY scaler scaler
+COPY tmp/2_fake-webserver.yaml 2_fake-webserver.yaml
+
+ENTRYPOINT ["/scaler", "scale","-f", "/2_fake-webserver.yaml"]
```

Comment:
We shouldn't be  copying 2_fake-webserver.yaml inside the image as this requires a new image on every config change. Lets research what are the options to mount these at runtime.
At the end the entrypoint should be only the binary and anything else should be passed at runtime.

Reply:
it will probably be for a separate PR, but at least lets research now and open an issue.

I am thinking maybe create a  config map directly from the file on github when running each test?
This is a bit prone to error when we update the paths, but would be easy to catch and fix.

Reply:
maybe also talk to the Prow team , they might give some other ideas.

Reply:
We can pass "scale","-f", "/2_fake-webserver.yaml" as k8s args.
Do you want the config map to be created automatically?
How about creating a config map with same contents as fake-webserver deployment in fake-webserver.yaml file

1 drawback will be having to change the config in 2 places.
But since they will be in the same file, maybe it won't be much of a problem

Reply:
https://gist.github.com/sipian/7f871178c186b9c966ecebcbb6685c05
An example config using this technique.